### ISSUE TITLE
Crashes with ambiguous error message

### ISSUE THREAD

**Issue by coolaj86** [letsencrypt/letsencrypt#540]
```bash
Traceback (most recent call last):
  File "./venv/bin/letsencrypt", line 9, in <module>
    load_entry_point('letsencrypt==0.1', 'console_scripts', 'letsencrypt')()
  File "/home/pi/lets-encrypt-preview/venv/local/lib/python2.7/site-packages/letsencrypt/cli.py", line 406, in main
    return args.func(args, config, plugins)
  File "/home/pi/lets-encrypt-preview/venv/local/lib/python2.7/site-packages/letsencrypt/cli.py", line 133, in auth
    plugins):
  File "/home/pi/lets-encrypt-preview/venv/local/lib/python2.7/site-packages/letsencrypt/client.py", line 205, in obtain_and_enroll_certificate
    cert, privkey, chain = self.obtain_certificate(domains, csr)
  File "/home/pi/lets-encrypt-preview/venv/local/lib/python2.7/site-packages/letsencrypt/client.py", line 153, in obtain_certificate
    authzr = self.auth_handler.get_authorizations(domains)
  File "/home/pi/lets-encrypt-preview/venv/local/lib/python2.7/site-packages/letsencrypt/auth_handler.py", line 79, in get_authorizations
    self._respond(cont_resp, dv_resp, best_effort)
  File "/home/pi/lets-encrypt-preview/venv/local/lib/python2.7/site-packages/letsencrypt/auth_handler.py", line 138, in _respond
    self._poll_challenges(chall_update, best_effort)
  File "/home/pi/lets-encrypt-preview/venv/local/lib/python2.7/site-packages/letsencrypt/auth_handler.py", line 189, in _poll_challenges
    "Failed Authorization procedure for %s" % domain)
letsencrypt.errors.AuthorizationError: Failed Authorization procedure for dangerpi.devices.coolaj86.com
```

Since this device is behind a proxy, I'm assuming that I need to specify some ports that should be used for communication, but `./venv/bin/letsencrypt --help` didn't give me such an option.

It takes minutes for the interface to load on Raspberry Pi, so I'll try the manual mode and see if that helps me.

**Comment by kuba** [comment]
This should be fixed by #536.

**Comment by coolaj86** [comment]
Yep, it is.
